even_fib_nums.js

```diff
@@ -1,35 +1,94 @@
+/*jshint esversion: 6 */
+
 /**
  * Return the total sum of all even fibonacci numbers up to and including
  * the value given to you at `maxFibValue`
  *
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
+
 function _sumFibs( maxFibValue ) {
   var sum = 0;
+  // var fibArr = [0,1];
+  // var evenFibArr = [];
 
-  // do your work here
+  let num1 = 0;
+  let num2 = 1;
+  let num3 = 0;
```

Comment:
This is a comment about your variables overall. I would suggest not using vars with numbers in them in the fashion on 'num1', 'num2', etc. This can become difficult to read on larger project and is not a good habit to pick up.